### PR TITLE
Updating singularity to 3.8.3

### PR DESCRIPTION
**Pull request by vsoch**
You can see the successful autamus build here: https://github.com/autamus/registry/pull/754

Signed-off-by: vsoch <vsoch@users.noreply.github.com>

### CODE REVIEW

File: var/spack/repos/builtin/packages/singularity/package.py [hunk 1]
```diff
@@ -22,11 +22,12 @@ class Singularity(SingularityBase):
 
     homepage = "https://singularity.hpcng.org/"
     url      = "https://github.com/hpcng/singularity/releases/download/v3.6.4/singularity-3.6.4.tar.gz"
-    git      = "https://github.com/hpcng/singularity.git"
+    url      = "https://github.com/hpcng/singularity/releases/download/v3.8.3/singularity-3.8.3.tar.gz"
```
Comment (haampie) [inline var/spack/repos/builtin/packages/singularity/package.py]:
```suggestion
    url      = "https://github.com/hpcng/singularity/releases/download/v3.8.3/singularity-3.8.3.tar.gz"
    git      = "https://github.com/hpcng/singularity.git"
```

**Review by haampie — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/singularity/package.py (hunk 1 above)
Comment (vsoch) [reply var/spack/repos/builtin/packages/singularity/package.py]: Weird I couldn't apply this change - but just pushed from local branch!

Comment (haampie) [reply var/spack/repos/builtin/packages/singularity/package.py]: Uh yeah, i also got the impression the suggestion didnt make sense, maybe a bug in github actually

**Review by haampie — APPROVED** [review]

**Comment by haampie** [comment]
@spackbot run pipeline

**Comment by haampie** [comment]
@spackbot run pipeline

**Pull request merged**